modin/__init__.py

```diff
@@ -53,7 +53,7 @@ def get_execution_engine():
             except ImportError:
                 pass
             else:
-                if version.parse(ray.__version__) != version.parse("0.8.6"):
+                if version.parse(ray.__version__) != version.parse("0.8.7"):
```

Comment:
@devin-petersohn you suggested relaxing ray version check in your comment:
https://github.com/modin-project/modin/issues/1958#issuecomment-680267928

How relaxed should the check be? Should we pin it to `0.8.x` or simply check the minimal version is equal and above pip version?

Reply:
Hi @binarycrayon thanks for this PR. Ray is going to release 1.0 in the next version. I would suggest pinging it to 0.8.7 for now and Modin can start testing against 1.0 release candidate.